core/src/main/java/com/hcl/ecomm/core/services/impl/CreateCartServiceImpl.java

```diff
@@ -47,16 +47,23 @@ public String getEmptyCartPath() {
 	
 
 	@Override
-	public JSONObject createGuestCart() {
+	public JSONObject createCart(String customerToken) {
 		LOG.debug("createGuestCart method start.");
```

Comment:
@DeepaliAgrwl : remove this log.debug from here

Reply:
Done